src/components/PublicMessage.jsx

```diff
@@ -28,15 +28,30 @@ export default function PublicMessage({message, routes}) {
           ))}
         </th>
       )}
-      <td colSpan={(routes) ? 1 : 2}>
+      <td
+        style={{
+          'border-right': `3px solid ${PRIORITY_COLORS[priority]}`,
+        }}
```

Comment:
This might be a more "classic web dev" opinion in the react context, but I would prefer to use css wherever possible over `style` attributes.

`.priority-x` classes could work for most of them where x is the number ?

Or we could do something like `PRIORITY_CLASSES = {'0': 'emergency', ...}` and use an enumerated string set for x ? (I think I'm less in favor of this if we're keeping this all at the component level though).

Reply:
Better semantically to do `.priority-{hmm}`, especially since the colors don't have anything to do with how the data is being *arranged*, just how it looks

Reply:
Yeah I would agree with that. My initial reservations were that I would also bubble that up all the way to the shape of the `usePublicMessages` return value.

I do already plan to introduce a formal "normalize" step in that hook as a part of #6 though and I can incorporate that there. Meaning

> `PRIORITY_CLASSES = {'0': 'emergency', ...}`

this can live in the component for now I think